subschemas/web2/src/config/index.ts

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+import development from './development';
+import production from './production';
+
+dotenv.config({ debug: true });
```

Comment:
we shouldn't need to run `dotenv` in here, it's already done in the `api-gateway` application